### DISCUSSION TITLE
Receiving Signal on CubeSatSim v1.3.2

### DISCUSSION THREAD

**Discussion by calebc1800** [alanbjohnston/CubeSatSim#319] [Q&A] [answered]
I currently have a CubeSatSim v1.3.2 running transmitting telemetry packets. I am looking to transmit a signal from the ground station for the CubeSatSim to receive. I have viewed the pcb layout and confirmed that the COM chip (SR105U) should be able to receive the signal. However, I am having trouble swapping the chip to listen for signal.  I am currently working on a script to trigger the PTT on the chip which is not working.

Has anyone been able to receive signal on their CubeSatSim using the SR105U?

**Comment by alanbjohnston** [comment, marked as answer]
Hi, the current beta software does support receiving a signal from the SR105U (SR_FRS_0W5).  Make sure you are running the latest by logging into your Pi which has internet connectivity:

`cd`

`CubeSatSim/update`

You can check to see if Command and Control is active by typing:

`CubeSatSim/config`

and looking for "Radio command and control is ON". If it says it is OFF, type this command to turn it on:

`CubeSatSim/config -T`

You can use the Command and Control part of the Test Plan to verify it is working:

https://github.com/alanbjohnston/CubeSatSim/wiki/v1.3.2-9.-Final-Testing#command-and-control-test

The current beta software just uses the SQUELCH to detect a command, so any carrier on the command and control frequency should trigger it.  You can see this squelch output in real-time if you connect a micro USB cable to the Pico and use the Arduino Serial Monitor to see the output, which will be updated every second as Squelch:1 (no signal) or Squelch:0 (signal received).

I am working on a software update that will allow PL tones to be used so that not any carrier will detect it.  I will announce it in this thread when it is ready to use.

I am also working on a hardware mod that will do DTMF tone detection and APRS packet decoding on the Pi Zero for command and control.  I will also announce it here.

If you are experimenting with the SR105U transceiver, if it is powered up (PD is 3V3) and PTT (Push To Talk) is not active (3V3), then  it is in receiver mode, and the SQ and AF_OUT will have signals.  If you want to receive weak signals, you will need to bypass the attenuator circuit, but for strong signals (such as a transmitter close to the CubeSatSim), then the attenuator doesn't cause any issues.

Good luck!

Alan
